docs/getting_started.md

```diff
@@ -67,6 +67,10 @@ Streamlit.
 
    Running a Streamlit app is no different than any other Python script.
    Whenever you need to view the app, you can use this command.
+   ```eval_rst
+   .. tip::
+      Did you know that you can pass either a local python script or a URL to `$ streamlit run` as an argument.
```

Comment:
Capitalize Python. End with question mark.

But actually I'd change this to something more like:

> Did you know you can also pass a URL to `streamlit run`? This is great when combined with Github Gists. For example:
>
> ```
> $ streamlit run https://raw.githubusercontent.com/streamlit/demo-uber-nyc-pickups/master/app.py
> ```

Reply:
We can make that work. This change is in the next push.